entrypoint.sh

```diff
@@ -2,4 +2,13 @@
 
 set -eo pipefail
 
-sh -c "dnscontrol $*"
+OUTPUT=`sh -c "dnscontrol $1"`
```

Comment:
```suggestion
OUTPUT="$(dnscontrol "$1")"
```

https://github.com/koalaman/shellcheck/wiki/SC2006

And I _believe_ it could do without `sh -c "..."` as  `$(...)`. I don't recall why the command has to evaluate in a sub shell.

Reply:
Interesting. I actually struggled with this, and I ended up using backticks here to because `"$(...)"` stripped all newlines in the output:

![image](https://user-images.githubusercontent.com/11269635/85271771-85622400-b47b-11ea-80a9-30532e66d381.png)

Bash/shell stuff is very much not my forté, so I'll check it out a bit and report back again! 🙂

Reply:
Yeah, that is because of field splitting (been there).

Try setting the `IFS` variable to empty on the line before `OUTPUT=""`:

`IFS=`

Reply:
Cool, that seems to do the trick! 😄